runtime/compiler/env/j9method.h

```diff
@@ -518,17 +525,21 @@ class TR_ResolvedJ9Method : public TR_J9Method, public TR_ResolvedJ9MethodBase
    virtual void makeParameterList(TR::ResolvedMethodSymbol *methodSym);
 
    /**
-    * @brief Check if a field is a QType or not.
+    * @brief Check if a field is with NullRestricted attribute.
     *
+    * @param[in] comp : The current compilation object
     * @param[in] cpIndex : the constant pool index of the field
+    * @param[in] isStatic: whether the field is static
+    * @param[in] isStore : whether or not to store into the field
```

Comment:
I think "whether or not to store into the field" would read more correctly as "whether the field is being used in the context of a store into the field," as the checking doesn't actually cause a store to be generated.

Reply:
Addressed in 32f6437